shared/config/resources.go

```diff
@@ -209,18 +191,16 @@ func newRocketPoolResources(network config.Network) *RocketPoolResources {
 		PreviousRewardsPoolAddresses: []common.Address{
 			common.HexToAddress("0x4d581a552490fb6fce5F978e66560C8b7E481818"),
 		},
-		PreviousProtocolDaoVerifierAddresses:   []common.Address{},
-		PreviousRocketNetworkPricesAddresses:   []common.Address{},
-		PreviousRocketNetworkBalancesAddresses: []common.Address{},
-		OptimismPriceMessengerAddress:          nil,
-		PolygonPriceMessengerAddress:           nil,
-		ArbitrumPriceMessengerAddress:          nil,
-		ArbitrumPriceMessengerAddressV2:        nil,
-		ZkSyncEraPriceMessengerAddress:         nil,
-		BasePriceMessengerAddress:              nil,
-		ScrollPriceMessengerAddress:            nil,
-		ScrollFeeEstimatorAddress:              nil,
-		RplTwapPoolAddress:                     hexToAddressPtr("0x7bb10d2a3105ed5cc150c099a06cafe43d8aa15d"),
+		PreviousProtocolDaoVerifierAddresses: []common.Address{},
+		OptimismPriceMessengerAddress:        nil,
```

Comment:
well this is just _looks_ confusing.

`nil` is a valid value for any slice type. Whatever references `PreviousProtocoLDaoVerifierAddresses` should be nil-safe.